### PR TITLE
Add allow_stock_to_be_exceeded flag in OrderFulfill mutation

### PR DESCRIPTION
**Pull request by L3str4nge**
Added `allow_stock_to_be_exceeded` flag in `OrderFulfill` mutation. This flag is set to `False` by default. If set to `True` then we can fulfillment an order when stock is out of quantity.  

In addition:
* added migration for `Stock` model for `quantity < 0` -> `saleor/warehouse/migrations/0015_alter_stock_quantity.py`
* fixed typo in module name -> `test_fulfullments_actions.py → test_fulfillments_actions.py`

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-4421 Extend OrderFulfill mutation by new flag to allow fulfill lines without stocks](https://app.clickup.com/t/2549495/SALEOR-4421)

File: saleor/graphql/order/mutations/fulfillments.py [hunk 1]
```diff
@@ -58,6 +58,11 @@ class OrderFulfillInput(graphene.InputObjectType):
         description="If true, send an email notification to the customer."
     )
 
+    allow_stock_to_be_exceeded = graphene.Boolean(
+        description="If true, then allow proceed fulfillment when stock is exceeded",
```
Comment (tomaszszymanski129) [inline saleor/graphql/order/mutations/fulfillments.py:62]: we end descriptions with dot 🚔

**Review by IKarbowiak — APPROVED** [review]

File: saleor/graphql/order/mutations/fulfillments.py (hunk 1 above)
Comment (L3str4nge) [reply saleor/graphql/order/mutations/fulfillments.py]: Fixed

**Review by tomaszszymanski129 — APPROVED** [review]

**Review by maarcingebala — APPROVED** [review]

**Review by korycins — COMMENTED** [review]
@L3str4nge  Before we will merge these changes, please create a PR to master and confirm that merged migrations will work well.

File: saleor/graphql/order/tests/test_fulfillment.py [hunk 2]
```diff
@@ -77,10 +77,122 @@ def test_order_fulfill(
         ]
     }
     mock_create_fulfillments.assert_called_once_with(
-        staff_user, None, order, fulfillment_lines_for_warehouses, ANY, True
+        staff_user, None, order, fulfillment_lines_for_warehouses, ANY, True, False
     )
 
 
+def test_order_fulfill_with_stock_exceeded_with_flag_disabled(
```
[the remark this answers is not in the document]
Comment (L3str4nge) [reply saleor/graphql/order/tests/test_fulfillment.py:84]: But it's a case when we have exceeded stock and disabled flag so maybe just `test_order_fulfill_with_stock_exceeded_and_flag_disabled` ?

[the remark this answers is not in the document]
Comment (korycins) [reply saleor/graphql/order/tests/test_fulfillment.py:84]: My bad, it is fine as it is now. Sorry!

**Review by korycins — APPROVED** [review]

**Review by korycins — APPROVED** [review]

**Review by eCzerniak — APPROVED** [review]

**Pull request merged**